src/components/navigation/NavigationSelectors.ts

```diff
@@ -0,0 +1,22 @@
+import * as _ from 'underscore';
+import {IReactVaporState} from '../../ReactVapor';
+import {PER_PAGE_NUMBERS} from './perPage/NavigationPerPage';
+
+export interface GetPaginationProps {
+    id: string;
+}
+
+const getPaginationPage = (state: IReactVaporState, props: GetPaginationProps): number => {
+    const pagination = _.findWhere(state.paginationComposite, {id: props.id});
+    return pagination && pagination.pageNb || 0;
+};
+
+const getPerPage = (state: IReactVaporState, props: GetPaginationProps): number => {
+    const pagination = _.findWhere(state.perPageComposite, {id: props.id});
+    return pagination && pagination.perPage || PER_PAGE_NUMBERS[1];

```

Comment:
why not PER_PAGE_NUMBERS[0] ?

Reply:
The default is `const defaultInitialPosition: number = Math.ceil(perPageNumber.length / 2) - 1;` from the NavigationPerPageConnected. I don't think it makes that much sense but I don't know how to do otherwise.

This will only apply if the pagination is not in the state, I could put 1 instead and it would still work

Reply:
If my memory is good, the default per page is 50, Its a good idea to set with 1.